examples/PromptInjectionDetector/PromptInjectionGPT4o.ipynb

```diff
@@ -0,0 +1,439 @@
+    "\n",
+    "\n",
+    "test_classify = classify_prompt(test_prompt)\n",
+    "print(\"Input:\", test_prompt)\n",
+    "print(\"Classified as:\", test_classify)\n",
+    "print(\"Score:\", 1 if test_classify == test_expected_label else 0)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Running across the dataset\n",
+    "\n",
+    "Now that we have automated classifying prompts, we can test the full set of articles using Braintrust's `Eval` function.\n",
+    "Behind the scenes, `Eval` will in parallel run the `classify_prompt` function on each article in the dataset, and then compare the results to the ground truth labels using a simple `NumericDiff` scorer. When it finishes running, it will print out the results with a link to Braintrust to dig deeper.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import braintrust\n",
+    "from autoevals import NumericDiff\n",
+    "from autoevals import Levenshtein \n",
+    "\n",
+    "braintrust.login(api_key=os.environ.get(\"BRAINTRUST_API_KEY\", \"<Your Braintrust API Key Here>\"))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {
+    "id": "Pd75YlQvw1K8"
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Experiment numericdiff_prompt_injection-1716236164 is running at https://www.braintrust.dev/app/braintrustdata.com/p/Prompt%20Injection%20Detection/experiments/numericdiff_prompt_injection-1716236164\n",
+      "Prompt Injection Detection (data): 30it [00:00, 42366.71it/s]\n"
+     ]
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "f58139264a7d49708c5fea62becb5aad",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Prompt Injection Detection (tasks):   0%|          | 0/30 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "=========================SUMMARY=========================\n",
+      "90.00% 'NumericDiff' score\n",
+      "\n",
+      "0.90s duration\n",
+      "\n",
+      "See results for numericdiff_prompt_injection-1716236164 at https://www.braintrust.dev/app/braintrustdata.com/p/Prompt%20Injection%20Detection/experiments/numericdiff_prompt_injection-1716236164\n"
+     ]
+    }
+   ],
+   "source": [
+    "eval_result = await braintrust.Eval(\n",
+    "    \"Prompt Injection Detection\",\n",
+    "    data=prompts_and_labels,\n",
+    "    task=classify_prompt,\n",
+    "    scores=[NumericDiff],\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "bAQ3gHsdh3JU"
+   },
+   "source": [
+    "## Quickly understand results with Braintrust's Experiment\n",
```

Comment:
I think title might be a bit off gramatically. Do you mean "Braintrust's experiment view"? Also why capitalize Experiment?